docs/user/integration/prometheus/README.md

```diff
@@ -0,0 +1,154 @@
+  - [Table of Content](#table-of-content)
+  - [Prerequisites](#prerequisites)
+  - [Context](#context)
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- The [Telemetry module](../../README.md) is added. For details, see [Quick Install](https://kyma-project.io/#/02-get-started/01-quick-install). <!-- This link differs for OS and SKR -->
+- If you want to use Istio access logs, make sure that the [Istio module](https://kyma-project.io/#/istio/user/README) is added.
+<!-- markdown-link-check-disable -->
+- Kubernetes CLI (kubectl) (see [Install the Kubernetes Command Line Tool](https://developers.sap.com/tutorials/cp-kyma-download-cli.html)).
+<!-- markdown-link-check-enable -->
+- UNIX shell or Windows Subsystem for Linux (WSL) to execute commands.
+
+> [!WARNING]
+- This guide describes a basic setup that you should not use in production. Typically, a production setup needs further configuration, like optimizing the amount of data to be collected and the required resource footprint of the installation. To achieve qualities like [high availability](https://prometheus.io/docs/introduction/faq/#can-prometheus-be-made-highly-available), [scalability](https://prometheus.io/docs/introduction/faq/#i-was-told-prometheus-doesnt-scale), or [durable long-term storage](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage), you need a more advanced setup.
+- This example uses the latest Grafana version, which is under AGPL-3.0 and might not be free of charge for commercial usage.
+
+## Context
+
+The Telemetry module supports shipping metrics from applications and the Istio service mesh to Prometheus using the OpenTelemetry protocol (OTLP). Prometheus is a widely used backend for collection and storage of metrics. The `kube-prometheus-stack` is a Helm chart which bundles Prometheus together with Grafana and the Alertmanager, to have an instant and comprehensive monitoring experience. Furthermore, it brings community-driven best practices on kubernetes monitoring including the node-exporter and kube-state-metrics component
+
+As the OpenTelemetry community is not that advanced yet in providing a full-blown Kubernetes monitoring, this guide shows how to combine the two worlds by integrating application and Istio metrics via the telemetry module, and the kubernetes monitoring via the features of the bundle.
+
+In this guide you will first deploy the `kube-prometheus-stack`, then you will configure the telemetry module to start metric ingestion, and then you deploy the sample application to illustrate custom metric consumption.
+
+![setup](./../assets/prometheus.drawio.svg)
+
+## Guide
+
+## Install the kube-prometheus-stack
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export K8S_PROM_NAMESPACE="{namespace}"
+    ```
+1. If you haven't created the Namespace yet, now is the time to do so:
```

Comment:
```suggestion
1. If you haven't created the namespace yet, now is the time to do so:
```
Would it make sense to switch the order of step 1 and 2? First check if you have the namespace and if you don't, create it. Then, export it as variable.

Reply:
If you have the namespace, you still need the variable as we use the variable in the later commands

Reply:
Pls confirm: I can perform step 1 even before a namespace exists?
Then how do I know what exactly to enter for the placeholder?

Reply:
The namespace you want to use (either existing or not) you should put in this env variable. Yes, you can do that at any time, setting just an env variable. And yes, you need to specify a namespace, there is no way around

Reply:
I see! And this typically happens (or should happen) before the namespace exists?

Reply:
The commands here are using the namespace name as variable input. Also the namespace creation will need the name. So you define once a variable and use it in all commands, including namespace creation